pyvista/core/utilities/geometric_sources.py

```diff
@@ -15,40 +15,65 @@
 from .helpers import wrap
 
 
-def translate(surf, center=(0.0, 0.0, 0.0), direction=(1.0, 0.0, 0.0)):
+def translate(
+    surf, center=(0.0, 0.0, 0.0), direction=(1.0, 0.0, 0.0), starting_direction=(1.0, 0.0, 0.0)
```

Comment:
Let's follow the name of  `pv.SolidSphere` .

```suggestion
    surf, center=(0.0, 0.0, 0.0), direction=(1.0, 0.0, 0.0), start_direction=(1.0, 0.0, 0.0)
```